patches/chromium/propagates_pointer_events_to_draggable_regions_in_frameless_windows.patch

```diff
@@ -0,0 +1,89 @@
++
+ @end
+ 
+ #endif  // COMPONENTS_REMOTE_COCOA_APP_SHIM_BRIDGED_CONTENT_VIEW_H_
+diff --git a/components/remote_cocoa/app_shim/bridged_content_view.mm b/components/remote_cocoa/app_shim/bridged_content_view.mm
+index fc9c7b30f0353451fdbe4676be5c90c3656b8109..6691609e06786a15afc399c0d8eba9dc13954e69 100644
+--- a/components/remote_cocoa/app_shim/bridged_content_view.mm
++++ b/components/remote_cocoa/app_shim/bridged_content_view.mm
+@@ -288,13 +288,15 @@ - (bool)needsUpdateWindows {
+ // could be optimized by telling the window server which regions should be
+ // instantly draggable without asking (tracked at https://crbug.com/830962).
+ - (NSView*)hitTest:(NSPoint)point {
++  return [super hitTest:point];
++}
++
++- (bool)isDraggableBackgroundAt:(NSPoint)point {
+   gfx::Point flippedPoint(point.x, NSHeight(self.superview.bounds) - point.y);
+   bool isDraggableBackground = false;
+   _bridge->host()->GetIsDraggableBackgroundAt(flippedPoint,
+                                               &isDraggableBackground);
+-  if (isDraggableBackground)
+-    return nil;
+-  return [super hitTest:point];
++  return isDraggableBackground;
+ }
+ 
+ - (void)processCapturedMouseEvent:(NSEvent*)theEvent {
+diff --git a/content/app_shim_remote_cocoa/render_widget_host_view_cocoa.h b/content/app_shim_remote_cocoa/render_widget_host_view_cocoa.h
+index 09b562ff00eb69beeab3332cfa0cd4b2005dda9e..3339187988296c6b03f8ff4aae4389d36a73b430 100644
+--- a/content/app_shim_remote_cocoa/render_widget_host_view_cocoa.h
++++ b/content/app_shim_remote_cocoa/render_widget_host_view_cocoa.h
+@@ -208,6 +208,8 @@ struct DidOverscrollParams;
+   base::scoped_nsobject<id> _accessibilityParent;
+ 
+   uint64_t popup_parent_ns_view_id_;
++
++  BOOL _isDragging;
+ }
+ 
+ @property(nonatomic, assign) NSRange markedRange;
+diff --git a/content/app_shim_remote_cocoa/render_widget_host_view_cocoa.mm b/content/app_shim_remote_cocoa/render_widget_host_view_cocoa.mm
+index 949f9a615d1842be4be496a4523411aa4e97a3a6..ad93ea0a2c41176db3f599695d14b0c4163044a5 100644
+--- a/content/app_shim_remote_cocoa/render_widget_host_view_cocoa.mm
++++ b/content/app_shim_remote_cocoa/render_widget_host_view_cocoa.mm
+@@ -162,6 +162,7 @@ - (BOOL)disableAutoHideCursor;
+ 
+ @interface NSView (ElectronCustomMethods)
+ - (BOOL)shouldIgnoreMouseEvent;
++- (BOOL)isDraggableBackgroundAt:(NSPoint)point;
+ @end
+ 
+ // RenderWidgetHostViewCocoa ---------------------------------------------------
+@@ -785,6 +786,20 @@ - (void)mouseEvent:(NSEvent*)theEvent {
+   else if (type == NSEventTypeLeftMouseUp)
+     _hasOpenMouseDown = NO;
+ 
++  if (type == NSEventTypeLeftMouseDown || type == NSEventTypeMouseMoved) {
++    NSView* contentView = [self.window contentView];
++    if ([contentView respondsToSelector:@selector(isDraggableBackgroundAt:)] &&
++        [contentView isDraggableBackgroundAt:[theEvent locationInWindow]]) {
++      _isDragging = YES;
++      [self.window performWindowDragWithEvent:theEvent];
```

Comment:
I believe the `performWindowDragWithEvent` only needs to be called for the mouse down event, the API will take care of the moved and mouse up events itself. Per document:

```
Your application (or a view) can call this method after receiving and examining a mouse-down event.
```

Reply:
true, good catch! [updated](https://github.com/electron/electron/pull/38208/commits/4ac2915e0296b05fd4277449674ecdf847fd33bf)